<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>572</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Rounded MT Bold</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SmartCalc_v1.0</string>
  </property>
  <property name="iconSize">
   <size>
    <width>21</width>
    <height>21</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(41, 50, 66);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(32, 39, 53);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(58, 70, 90);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(50, 61, 79)
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(58, 70, 90);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(50, 61, 79)
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(58, 70, 90);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(50, 61, 79)
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>140</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(58, 70, 90);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(50, 61, 79)
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(58, 70, 90);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(50, 61, 79)
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>370</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(58, 70, 90);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(50, 61, 79)
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(58, 70, 90);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(50, 61, 79)
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(58, 70, 90);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(50, 61, 79)
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(58, 70, 90);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(50, 61, 79)
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(58, 70, 90);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(50, 61, 79)
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>250</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color:rgb(114, 215, 151);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(104, 196, 138);
}</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color:rgb(114, 215, 151);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(104, 196, 138);
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color:rgb(114, 215, 151);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(104, 196, 138);
}</string>
    </property>
    <property name="text">
     <string>−</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>430</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color:rgb(114, 215, 151);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(104, 196, 138);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(41, 50, 66);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(32, 39, 53);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calc">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color:rgb(114, 215, 151);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(104, 196, 138);
}</string>
    </property>
    <property name="text">
     <string extracomment="Calculate">=</string>
    </property>
    <property name="iconSize">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_bracket">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(41, 50, 66);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(32, 39, 53);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close_bracket">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(41, 50, 66);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(32, 39, 53);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>700</width>
      <height>84</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>11</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>42</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: disable;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>700</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: disable;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>200</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	border : 2px solid rgb(41, 50, 66);
	background : rgb(58, 70, 90);
}</string>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x = </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete_prev">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(41, 50, 66);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(32, 39, 53);
}</string>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color: rgb(41, 50, 66);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(32, 39, 53);
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
